src/components/SignUpForm/styles.scss

```diff
@@ -0,0 +1,37 @@
+@import "../../styles/main.scss";
+
+.container {
+    min-height: 100vh;
+    @include flex-center();
+}
```

Comment:
this `.container` class selector is not being used in this component but in `SignUp` instead. Do you think we should move it there?